### PR TITLE
Adding metrics for limits in CPU, memory, and disk

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

**Comment by vishh** [comment]
Ping @rjnagal.

File: sinks/api/supported_metrics.go [hunk 1]
```diff
@@ -86,6 +102,22 @@ var statMetrics = []SupportedStatMetric{
 	},
 	{
 		MetricDescriptor: MetricDescriptor{
+			Name:        "memory/limit",
+			Description: "Memory limit in bytes",
```
Comment (rjnagal) [inline sinks/api/supported_metrics.go]: remove units as we have a separate field for it.

File: sinks/api/supported_metrics.go [hunk 2]
```diff
@@ -198,6 +230,31 @@ var statMetrics = []SupportedStatMetric{
...
+		},
+		HasValue: func(spec *cadvisor.ContainerSpec) bool {
+			return spec.HasFilesystem
+		},
+		GetValue: func(spec *cadvisor.ContainerSpec, stat *cadvisor.ContainerStats) []internalPoint {
+			result := make([]internalPoint, 0, len(stat.Filesystem))
+			for _, fs := range stat.Filesystem {
+				result = append(result, internalPoint{
+					value: int64(fs.Limit),
+					labels: map[string]string{
```
Comment (rjnagal) [inline sinks/api/supported_metrics.go]: How do we handle multiple fs?

Comment (vishh) [inline sinks/api/supported_metrics.go]: We use device id's as a label to differentiate between filesystems.

File: sinks/api/supported_metrics.go (hunk 1 above)
Comment (vishh) [inline sinks/api/supported_metrics.go]: Done

**Comment by rjnagal** [comment]
LGTM. Needs rebase to remove other patches.

**Comment by vishh** [comment]
Fixed the commit issues. PTAL @rjnagal

**Comment by rjnagal** [comment]
LGTM. waiting for travis.

**Comment by vishh** [comment]
@rjnagal: I fixed the e2e. This PR is good to go now. Jenkins would have covered unit testing as well.

**Pull request merged**